### PR TITLE
Starting new tutorial on analysis of Epigenome-wide association studies (EWAS)

### PR DESCRIPTION
**Pull request by kpbioteam**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -93,5 +93,8 @@ DEPENDENCIES
   jekyll-feed!
   jemoji!
 
+RUBY VERSION
```
Comment (bgruening) [inline Gemfile.lock]: Please don't commit this file. You can remove it from git with `git rm`.

File: package.json [hunk 2]
```diff
@@ -12,6 +12,6 @@
   },
   "homepage": "https://github.com/galaxyproject/training-material#readme",
   "dependencies": {
```
Comment (bgruening) [inline package.json]: this files as well is not needed, and the package-lock as well.

File: templates/tutorials/ewas-suite/data-library.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+libraries:
+    # This needs to be changed to reference your Zenodo data
+    - name: "Small test files"
```
Comment (bgruening) [inline templates/tutorials/ewas-suite/data-library.yaml]: Here you need to include your Zenodo files.

File: templates/tutorials/ewas-suite/data-manager.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# configuration for fetch and index genomes
+---
+data_managers:
```
Comment (bgruening) [inline templates/tutorials/ewas-suite/data-manager.yaml]: This file is probably not needed in your case as you don't have a data manager.

File: templates/tutorials/ewas-suite/slides.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+layout: introduction_slides
```
Comment (bgruening) [inline templates/tutorials/ewas-suite/slides.html]: Do you plan to add slides?

File: templates/tutorials/ewas-suite/tools.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (bgruening) [inline templates/tutorials/ewas-suite/tools.yaml]: remove, this entire folder can be removed.

File: templates/tutorials/ewas-suite/workflows/empty_workflow.ga [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+    "a_galaxy_workflow": "true",
```
Comment (bgruening) [inline templates/tutorials/ewas-suite/workflows/empty_workflow.ga]: remove

File: topics/epigenetics/metadata.yaml [hunk 8]
```diff
@@ -54,6 +54,25 @@ material:
...
+    title: "EWAS data analysis"
+    type: "tutorial"
+    name: "ewas-suite"
+    zenodo_link: "https://zenodo.org/record/1251211#.WwREQ1Mvz-Y"
+    galaxy_tour: yes
+    hands_on: yes
+    slides: yes
+    workflows: yes
+    questions:
+      - "What is methylation and why it cannot be recognised by a normal NGS procedure?"
```
Comment (bgruening) [inline topics/epigenetics/metadata.yaml]: .... by normal DNA sequencing?

File: topics/epigenetics/metadata.yaml [hunk 9]
```diff
@@ -54,6 +54,25 @@ material:
...
+      - "Can a different methylation influence the expression of a gene? How?"
+      - "Which tools you can use to analyse methylation data?"
```
Comment (bgruening) [inline topics/epigenetics/metadata.yaml]: You should probably mention the use of 450k data. Maybe even in the title?

File: topics/epigenetics/tutorials/ewas-suite/data-library.yaml [hunk 10]
```diff
@@ -0,0 +1,7 @@
+---
+libraries:
+    # This needs to be changed to reference your Zenodo data
+    - name: "Small test files"
+      files:
+          - url: "http://raw.githubusercontent.com/bgruening/galaxytools/master/tools/rna_tools/sortmerna/test-data/read_small.fasta"
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/data-library.yaml]: Add your Zenodo files here.

File: topics/epigenetics/tutorials/ewas-suite/data-manager.yaml [hunk 11]
```diff
@@ -0,0 +1,18 @@
+# configuration for fetch and index genomes
+---
+data_managers:
+    # Data manager ID
+    - id: url to data manager on ToolShed
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/data-manager.yaml]: remove

File: topics/epigenetics/metadata.yaml (hunk 8 above)
Comment (kpbioteam) [reply topics/epigenetics/metadata.yaml]: Hi Bjorn,  I didn't save a changes when I was editing copied example. 
Now I've provide my questions and objectives, thanks for spotting this.

File: templates/tutorials/ewas-suite/slides.html (hunk 5 above)
Comment (kpbioteam) [reply templates/tutorials/ewas-suite/slides.html]: Yes, I would like to provide an introduction for the topic

**Comment by kpoterlowicz** [comment]
hi could you label it as work in progress, as it is still under active development

**Comment by bebatut** [comment]
Done @kpoterlowicz. Let us know when it is ready to review 😄

**Comment by kpoterlowicz** [comment]
thanks @bebatut  will do :)

**Pull request closed without merging**